draft-dnsop-deleg.md

```diff
@@ -263,53 +232,67 @@ Some zone owners may wish to use multiple providers to serve their zone, in whic
 
-If a zone operator removes all NS records before DELEG and SVCB records are implemented by all clients, the availability of their zones will be impacted for the clients that are using non-supporting resolvers. In some cases, this may be a desired quality, but should be noted by zone owners and operators.
+    config2.example.net 3600    IN SVCB . (
+                    ipv4hint=192.0.2.54,192.0.2.56
+                    ipv6hint=2001:db8:2423::3,2001:db8:2423::4 )
 
-## Response Size Considerations
+The above records indicate to the client that the actual configuration for the example.com zone can be found at config2.example.net
 
-For latency-conscious zones, the overall packet size of the delegation records from a parent zone to child zone should be taken into account when configuring the NS, DELEG and SVCB records. Resolvers that wish to receive DELEG and SVCB records in response SHOULD advertise and support a buffer size that is as large as possible, to allow the authoritative server to respond without truncating whenever possible.
+Later sections of this document will go into more detail on the resolution process using these records.
 
 
 # Implementation
 
 This document introduces the concept of signaling capabilities to clients on how to connect and validate a subdomain. This section details the implementation specifics of the DELEG record for various DNS components.
 
-## Including DELEG RRs in a Zone
+## Deployment Considerations
 
-A DELEG RRset MAY be present at a delegation point.  The DELEG RRset MAY contain multiple records. DELEG RRsets MUST NOT appear at a zone's apex.
+The DELEG and SVCB records are intended to replace the NS record while also adding additional functionality in order to support additional transports for the DNS. Below are discussions of considerations for deployment.
 
-A DELEG RRset MAY be present with or without NS or DS RRsets at the delegation point. 
+### AliasMode and ServiceMode in the Parent
 
-Construction of a DELEG RR requires knowledge which implies communication between the
-operators of the child and parent zones. This communication is an operational matter not covered by this document.
+Both the AliasMode and ServiceMode records can be returned for the DELEG record from the parent. This is different from the SCVB  {{?RFC9460}} specification and only applies for the DELEG RRSet in the parent.
 
-### Signing DELEG RRs
 
-A DELEG RRset MUST be DNSSEC signed if the zone is signed.
+### Rollout
+
+When introduced, the DELEG and SVCB records might not initially be supported by the DNS root or TLD operators. Zone owners may place these records into their zones before the zones above them have done so. However, doing so is only useful for further delegations down the tree as an SVCB record at the zone apex alone does not indicate a new delegation type. The only way to discover new delegations is with the DELEG record at the parent.
```

Comment:
"Zone owners may place these records into their zones before the zones above them have done so."

Zone owners should always place the records in the child zone before the parent or should would end up having a lame DELEG record at the parent.
A side-note is that it will not be effective until the parent supports DELEG and implements the record.

Reply:
Rewrote that section a bit to address it.